<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Вращение осей</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="rotateAxisX">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAxisX">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="rotateAxisY">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAxisY">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="rotateAxisZ">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelAxisZ">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Сбросить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>A</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinA_x">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-400</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="stepType">
            <enum>QAbstractSpinBox::DefaultStepType</enum>
           </property>
           <property name="value">
            <number>-150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinA_y">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-400</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>-150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinA_z">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-150</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>B</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinB_x">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-400</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>-150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinB_y">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinB_z">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-150</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>C</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinC_x">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinC_y">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-400</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>-150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinC_z">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-150</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>D</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinD_x">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinD_y">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
           <property name="value">
            <number>150</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinD_z">
           <property name="minimumSize">
            <size>
             <width>52</width>
             <height>0</height>
            </size>
           </property>
           <property name="minimum">
            <number>-150</number>
           </property>
           <property name="maximum">
            <number>400</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="drawArea" native="true">
     <property name="minimumSize">
      <size>
       <width>550</width>
       <height>550</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
